### PR TITLE
Features/add qualify to app review #167549021

### PR DESCRIPTION
**Pull request by akegan**
[ticket](https://www.pivotaltracker.com/story/show/167549021)

Still pending:

- [x] -  Finalize content/layout
- [x] - Move english content into locale-en.json when finalized

Open questions:
- Should I move lending agent/lending institution helper functions into directive similar to [preferencesSummaryDirective](https://github.com/Exygy/sf-dahlia-web/blob/master/app/assets/javascripts/short-form/directives/preferencesSummaryDirective.js.coffee)?
- Alternatively, should I move some of the functionality from the [lendingInstitution controller](https://github.com/Exygy/sf-dahlia-web/blob/master/app/assets/javascripts/short-form/components/lendingInstitution.js.coffee) to short form controller?

### CODE REVIEW

**Comment by akegan** [comment]
@callaghanc here are my changes so far, still not 100% convinced that it's the right place to put the functions since the short form controller is such an enormous fn and we just did a bunch of refactoring to try to pull things out of it if possible.

**Comment by akegan** [comment]
@callaghanc just pushed up changes with some tests. Let me know if that's what you had in mind, and if there are other functions you were thinking needed test coverage

**Comment by callaghanc** [comment]
@akegan looks good

**Comment by akegan** [comment]
ok are we good to merge then? I can do another round of QA before merging, but I think I need you to approve since I own the PR

**Comment by callaghanc** [comment]
ah right makes sense, i'll add an approving review. are you satisfied with the layout of the qualify section? that was listed as pending above

**Comment by akegan** [comment]
yep! anita our content designer signed off on my design this morning, just forgot to update the PR with that info

**Comment by akegan** [comment]
@callaghanc just updated with moving content to locale-en.json. Can you check out those changes too? I'm wondering if I should move some of them to the component instead of the slim template

File: app/assets/json/translations/locale-en.json [hunk 1]
```diff
@@ -565,6 +566,7 @@
         "PLEASE_SELECT_ONE": "Please select one.",
         "PLEASE_SELECT_PREFERENCE": "If you have one of these lottery preferences, select it below:",
         "PREAPPROVAL_LETTER": "Pre-approval letter",
+        "PREAPPROVAL_LETTER_ATTACHED": "Pre-approval letter attached",
```
Comment (akegan) [inline app/assets/json/translations/locale-en.json]: could probably use the file_attached if I can put translated "Pre-approval" in there too.

Comment (akegan) [reply app/assets/json/translations/locale-en.json]: ended up moving this to the component

**Pull request merged**